internal/setup/setup.go

```diff
@@ -831,10 +830,12 @@ func pathInfoToYAML(pi *PathInfo) (*yamlPath, error) {
 	case CopyPath:
 		path.Copy = pi.Info
 	case TextPath:
-		path.Text = &pi.Info
+		text := pi.Info
+		path.Text = &text
```

Comment:
Discussed offline: Change the function comment to add `the returned object is going to have pointers to the given PathInfo object`. We should change the for loop and add `info := info` to avoid the mutation plus a TODO comment to remove it once we update to Go 1.22.

Reply:
Changed per suggestion. Thanks!